src/videojs-http-streaming.js

```diff
@@ -509,6 +509,36 @@ class HlsHandler extends Component {
     }
     super.dispose();
   }
+
+  getPlaybackStats() {
+    const getTimeRanges = (player, type) => {
+      const timeRangesList = [];
+      const timeRanges = player[type]();
+
+      for (let i = 0; i < timeRanges.length; i++) {
+        timeRangesList.push({
+          start: timeRanges.start(i),
+          end: timeRanges.end(i)
+        });
+      }
+
+      return timeRangesList;
+    };
+
+    return {
+      videoPlaybackQuality: this.tech_.getVideoPlaybackQuality(),
+      playerDimensions: this.tech_.currentDimensions(),
+      vhsStats: this.stats,
```

Comment:
This seems a bit ambiguous in naming considering we have `vhsStats` as a sub property of `playbackStats`

Reply:
Do we want to merge with `stats` or perhaps turn `stats` into a function that has all this information?